simpeg_drivers/components/factories/survey_factory.py

```diff
@@ -375,7 +375,7 @@ def _tdem_arguments(self, data=None, local_index=None, mesh=None):
         tx_list = []
         rx_factory = ReceiversFactory(self.params)
         tx_factory = SourcesFactory(self.params)
-        for tx_locs, rx_ids in zip(tx_locs, rx_lookup):
+        for tx_locs, rx_ids in zip(tx_locs, rx_lookup, strict=False):
```

Comment:
**question:** strict=True? is it expected to always have len() == len()?

@domfournier  @benk-mira  ?